packages/headless/src/modules/brands/validators/brand.create.validator.ts

```diff
@@ -0,0 +1,21 @@
+import * as Joi from 'joi';
+
+import { Brand } from './../../../entity/brand';
+
+export const BrandCreateSchema = Joi.object<Brand>({
+    id: Joi.string(),
+    info:{
```

Comment:
please ensure the object info is required or not.

Reply:
as per the previous project, only the name is required

Reply:
if you don't put info.name  Then it ignores the name property. Please test this

Reply:
You sent
if you don't put info.name Then it ignores the name property. Please test this

Reply:
no it doesnt

Reply:
I test it..it doesn't work.